src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/EditorTestCompositions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Razor.Test.Common;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Test.Common
+{
+    public static class EditorTestCompositions
```

Comment:
Moved this into its own class because the Razor.LanguageServerClient test project had trouble loading the Microsoft.VisualStudio.Text.Implementation assembly (and we also don't need to load it there)